dashboard/app/views/levels/_teacher.html.haml

```diff
@@ -63,25 +68,26 @@
             .level
               %button.btn.btn-info#clear-response{"data-user-level-url" => url_for(contained_user_level)}
                 = t('teacher.panel.clear_response')
-        .level
-          - if @level && @level.ideal_level_source_id && @script # 'solutions' for blockly-type levels
-            - if @ideal_level_source
-              %h4= t('teacher.panel.viewing_solution')
-            - else
-              = link_to build_script_level_path(@script_level, {solution: true}.merge(@section ? {section_id: @section.id} : {})) do
-                %button.btn.btn-info= t('reference_area.teacher')
-          - if @level.try(:examples).present? && current_user.try(:authorized_teacher?) # 'solutions' for applab-type levels
-            - @level.examples.each_with_index do |example, index|
-              - example_link = send("#{@level.game.app}_project_view_projects_url".to_sym, channel_id: example, host: 'studio.code.org', port: 443, protocol: :https)
-              = link_to example_link, target: "_blank" do
-                %button.btn.btn-info
-                  - if @level.examples.many?
-                    = t('teacher.panel.example_with_number', solution_number: index + 1)
-                  - else
-                    = t('teacher.panel.example')
-          -# Get rid of this once we turn on viewAsToggle experiment
-          = link_to build_script_level_path(@script_level, @section? {section_id: @section.id} : {}) do
-            %button.btn.btn-info{id: "try-it-yourself"}= t('teacher.panel.try_it_yourself')
+        - if @level
+          .level
+            - if @level && @level.ideal_level_source_id && @script # 'solutions' for blockly-type levels
```

Comment:
nit: you could eliminate a few checks here (`if @level && `) and below (`@level.try`) now that you're protecting the entire block with `if @level`